lager/extra/qt.hpp

```diff
@@ -0,0 +1,85 @@
+//
+// lager - library for functional interactive c++ programs
+// Copyright (C) 2017 Juan Pedro Bolivar Puente
+//
+// This file is part of lager.
+//
+// lager is free software: you can redistribute it and/or modify
+// it under the terms of the MIT License, as detailed in the LICENSE
+// file located at the root of this source code distribution,
+// or here: <https://github.com/arximboldi/lager/blob/master/LICENSE>
+//
+
+#pragma once
+
+#include <lager/cursor.hpp>
+#include <lager/reader.hpp>
+#include <lager/watch.hpp>
+
+namespace lager {
+
+namespace detail {
+
+struct qt_helper
+{};
+
+} // namespace detail
+
+/*!
+ * Use in a class declaration before any FunQ definition.
```

Comment:
Is this still true? I can't see you using it anywhere!

Reply:
Nope :p Good catch!